src/coord/src/session.rs

```diff
@@ -385,15 +385,42 @@ pub enum PortalState {
     NotStarted,
     /// Portal is a rows-returning statement in progress with 0 or more rows
     /// remaining.
-    InProgress(Option<RowBatchStream>),
+    InProgress(Option<InProgress>),
     /// Portal has completed and should not be re-executed. If the optional string
     /// is present, it is returned as a CommandComplete tag, otherwise an error
     /// is sent.
     Completed(Option<String>),
 }
 
-/// A stream of batched rows.
-pub type RowBatchStream = Box<dyn Stream<Item = Vec<Row>> + Send + Unpin>;
+/// State of an in-progress, rows-returning portal.
+pub struct InProgress {
```

Comment:
Out of situ this struct name/field combo is pretty ambiguous--maybe throw `rows` on either the struct or the field names?

Reply:
Yes please!